### ISSUE TITLE
Input Reads or Bins

### ISSUE THREAD

**Issue by colbyga** [geronimp/graftM#253]
Hi, 

I am trying to use graftM for the first time. I have paired-end shotgun metagenomic data for 8 samples. 

1) Should I submit my qc'd reads or refined bins as input to build a phylogenetic tree? As far as I can see I need to input my unassembled reads qc'd. Is this correct?

2) Is it possible to run graft on more than one "--graftm_package" at a time? eg. could I run it on all of the Ribosomal proteins in the list (https://data.ace.uq.edu.au/public/graftm/7/)?

**Comment by wwood** [comment]
Hi,

>Should I submit my qc'd reads or refined bins as input to build a phylogenetic tree? As far as I can see I need to input my unassembled reads qc'd. Is this correct?

Do you mean to classify the reads or to build a gpkg? You can run `graft` on reads or contigs/genomes, though you may want to input translated proteins instead to reduce noise, particularly since the ORF calling program OrfM is tuned for reads.

>Is it possible to run graft on more than one "--graftm_package" at a time? eg. could I run it on all of the Ribosomal proteins in the list (https://data.ace.uq.edu.au/public/graftm/7/)?

Right now I'm afraid only a single gpkg can be run at a time, though several search HMMs can be applied at once.

Hope that helps, ben

**Comment by colbyga** [comment]
Hi Ben, thanks for the advice! 

I want to classify the reads/contigs and place them in a tree, I wasn't trying build a gpkg. 

However, do you think it would be possible or ill advised to concatenate the 20 or so ribosomal fasta files and build a gpkg. Is there a reason not to do this? Is it because I wouldn't be able to have an alignment?

**Comment by wwood** [comment]
Right, the alignment at the junction of the proteins in the multiple sequence alignment might lead to artificial results. If you are interested in building a phylogenetic tree from ribosomal proteins, you might be interested in https://github.com/Ecogenomics/GTDBtk or if you want to link the contigs to reads using ribosomal proteins https://github.com/wwood/singlem

ben
